### ISSUE TITLE
Constantly "Indexing source files"

### ISSUE THREAD

**Issue by inspiredearth** [HvyIndustries/crane#382]
Is it normal that Crane constantly has a message in the status bar of Visual Code saying "Indexing source files" ? It's always there, and seems to me it shouldn't be. Either it indexes the files, and is done. Or there is nothing to index, and it doesn't do anything.

**Comment by nevadascout** [comment]
It sounds like a bug to me, can you share more about your setup? Does your workspace have any symlinks, network drives, etc?

**Comment by jhard** [comment]
As I've just encountered the same: it appears to happen when you don't have a workspace set up.
Setting up a workspace and adding a folder leads to it showing the progress and eventually switching to the flame and Crane + version.

**Comment by inspiredearth** [comment]
Yes, that sounds about right. In my situation there was no work space set up

On 1 May 2018 7:16:51 AM jhard <notifications@github.com> wrote:
As I've just encountered the same: it appears to happen when you don't have 
a workspace set up.
Setting up a workspace and adding a folder leads to it showing the progress 
and eventually switching to the flame and Crane + version.—
